### DISCUSSION TITLE
Empty language card

### DISCUSSION THREAD

**Discussion by burhwn** [anuraghazra/github-readme-stats#1988] [Q&A] [answered]
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=burhanyasinkose&langs_count=8&count_private=false&layout=compact&theme=react&hide_border=true&bg_color=0D1117)](https://github.com/anuraghazra/github-readme-stats)

Dont Work Why?
![image](https://user-images.githubusercontent.com/75111410/187511302-590c78d4-ed51-4be6-be2c-96d8eb54f8d7.png)

**Comment by rickstaa** [comment, marked as answer]
@burhanyasinkose Thanks for your question. The most likely answer is that we currently count forks, and the public Vercel instance does not have access to private repositories. Please see https://github.com/anuraghazra/github-readme-stats/discussions/1910 for more information. I hope that helps.
